README.md

```diff
@@ -250,4 +366,4 @@ The automatized workflow is based on:
 [3] Andre Santos Ribeiro, Luis Miguel Lacerda, Hugo Alexandre Ferreira, Multimodal Imaging Brain Connectivity Analysis (MIBCA) toolbox, 2015 Jul 14. [Available here](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4511822/)
 
 
-[![Build Status](https://travis-ci.org/the-virtual-brain/tvb-recon.svg?branch=master)](https://travis-ci.org/the-virtual-brain/tvb-recon) [![Coverage Status](https://coveralls.io/repos/github/the-virtual-brain/tvb-recon/badge.svg)](https://coveralls.io/github/the-virtual-brain/tvb-recon)
+[![Build Status](https://img.shields.io/badge/Build-error-critical)](https://travis-ci.org/the-virtual-brain/tvb-recon) [![Coverage Status](https://img.shields.io/badge/Coverage-unknown-informational)](https://coveralls.io/github/the-virtual-brain/tvb-recon)
```

Comment:
Forgot to comment on this change. Isn't this a link to a static badge? 
We need to show here the badge generated by travis for each build, so, I believe this link is correct: https://travis-ci.org/the-virtual-brain/tvb-recon.svg , but maybe we could remove: branch=master ?

Reply:
Yes, my bad, It should be a dynamic badge. I'll revert these changes. 
https://travis-ci.org/the-virtual-brain/tvb-recon.svg -  displays "build failing"
https://travis-ci.org/the-virtual-brain/tvb-recon.svg?branch=master - displays "build error"
In my opinion, I think we should use the link without branch=master.